api_etl/services/transport/mot.py

```diff
@@ -76,13 +82,19 @@ def transform(self, service_manifest, input_dir, output_dir):
                 print " ... skipping in DEV mode"
                 return output_dir
 
-            encoding_check = chardet.detect(open(filename).read())
-            self.encoding = encoding_check.get('encoding', 'windows-1252')
+            detector = UniversalDetector()
+            with opener(filename) as input_file:
+                for line in input_file.readlines():
+                    detector.feed(line)
+                    if detector.done:
```

Comment:
Thinking about this it might not be the best because there might be a stray character on the last line?